scripts/darwin/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Load system env only
+source /etc/bashrc
+source /etc/profile
+
+# Make sure home is pointing to current directory
+export HOME=$(pwd)
+
+# Download spack
+if [ ! -d "spack" ]; then
+  git clone https://github.com/spack/spack.git
+fi
+
+# Initialize spack env
+. spack/share/spack/setup-env.sh
+
+# Create .spack folder
+if [ ! -d ".spack" ]; then
+  mkdir -p .spack
+fi
+
+# Setup spack package yaml
+echo "packages:" > .spack/packages.yaml
```

Comment:
Maybe this could just be a separate file in source control and you just copy it?

Reply:
I have made it dependent on whatever, packages are specified with the env variables in the ci. It is more clear why I took this approach now. But really it doesn't matter one could simply define every module that might be needed in a single file... I'm not sure how much it matters.